src/include/OpenImageIO/texture.h

```diff
@@ -264,6 +266,8 @@ class OIIO_API TextureOpt {
     float rblur;   ///< Blur amount in the r direction
     float rwidth;  ///< Multiplier for derivatives in r direction
 
+    int colortransformid;       ///< Color space id of the texture
```

Comment:
If colortransformid can never be negative, why not make it an unsigned int?

Reply:
Just general animus toward unsigned types? All the other fields in this struct were int even though they technical have no meaningful negative values.